### ISSUE TITLE
Building for target Android 12

### ISSUE THREAD

**Issue by hypnocill** [vidit-me/react-native-jitsi-meet#3]
When I build for Android 12, I get error:
Manifest merger failed : Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined.

A workaround seems to be "tools:node="replace"" in the app's AndroidManifest. The app builds but this seems to cause issues with other libraries and push notifications.

Do you have any idea on how to build for Android 12 without using "tools:node="replace""?

**Comment by TacticCoder** [comment]
@hypnocill We will check this soon.

**Comment by hypnocill** [comment]
> @hypnocill We will check this soon.

thank you

**Comment by hypnocill** [comment]
@TacticCoder After removing "tools:node="replace" this exact problem for setting "android:exported" seems to be caused by another library I've used. So I changed that to a newer version and added "android:allowBackup="false" as described in the docs in this repo, then I got:
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:16:7-34
  	is also present at [org.jitsi.react:jitsi-meet-sdk:5.1.0] AndroidManifest.xml:32:9-35 value=(true).
which I managed to fix by adding "tools:replace="android:allowBackup""

The build was successful and on a brief test it looks like Jitsi works, push notifications work too now.

Thank you for the response and sorry for bothering you
